tools/test-npm.sh

```diff
@@ -3,11 +3,11 @@
 set -e
 
 # always change the working directory to the project's root directory
-cd $(dirname $0)/..
+cd $(dirname ${BASH_SOURCE[0]})/..
```

Comment:
Seems zsh doesn't sport this variable unfortunately, There is an [alternative](http://stackoverflow.com/a/23259585/808699), but I'd rather avoid using shell-specific things.